### PR TITLE
Add the lock screen time function after updating and activating the screen save

### PR DESCRIPTION
**Pull request by zhuyaliang**

### CODE REVIEW

File: src/mate-screensaver-preferences.c [hunk 1]
```diff
@@ -1067,8 +1101,9 @@ static char *
 format_value_callback_time (GtkScale *scale,
                             gdouble   value)
 {
+	/*You need to make up for 27 characters in length, otherwise the display will split into different lines*/
 	if (value == 0)
-		return g_strdup_printf (_("Never"));
+		return g_strdup_printf (_("Never                      "));
```
[the remark this answers is not in the document]
Comment (cwendling) [reply src/mate-screensaver-preferences.c:1106]: BTW, at least use something like `g_strdup_printf ("%-27s", _("Never"));`: it's both clearer and doesn't make a silly translatable string.

**Review by cwendling — CHANGES_REQUESTED** [review]

File: src/mate-screensaver-preferences.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhuyaliang) [reply src/mate-screensaver-preferences.c:1106]: This is indeed a foolish code. Thank you for reminding me to use ```g_strdup_printf ("% -27s", _ ("Never"))```

**Comment by cwendling** [comment]
@zhuyaliang I ended up making quite a few changes, see https://github.com/mate-desktop/mate-screensaver/compare/lock-delay...cwendling:mate-screensaver:lock-delay
What do you think of those?

**Comment by zhuyaliang** [comment]
@cwendling It looks great. Please merge this PR and initiate a new one. I will review the code as soon as possible

**Comment by cwendling** [comment]
> Please merge this PR and initiate a new one.

merge this one first, or replace it with one containing the fixes?

**Comment by zhuyaliang** [comment]
My suggestion is to merge this first, and then review and merge your repair PR

**Comment by zhuyaliang** [comment]
@cwendling 
> I ended up making quite a few changes, see https://github.com/mate-desktop/mate-screensaver/compare/lock-delay...cwendling:mate-screensaver:lock-delay
What do you think of those?

This is your labor success, I need your consent

**Pull request merged**